pkg/operator/configobservation/images/observe_images.go

```diff
@@ -56,3 +59,117 @@ func ObserveInternalRegistryHostname(genericListers configobserver.Listers, reco
 	}
 	return observedConfig, errs
 }
+
+func ObserveExternalRegistryHostnames(genericListers configobserver.Listers, recorder events.Recorder, existingConfig map[string]interface{}) (map[string]interface{}, []error) {
```

Comment:
nit: missing godoc

Reply:
you pass the event recorder but I don't see any events recorded in this observer... you probably want to include event when you observe a change in external registry hostname?